client/my-sites/site-settings/speed-up-site-settings.jsx

```diff
@@ -41,28 +81,59 @@ class SpeedUpSiteSettings extends Component {
 			isSavingSettings,
 			translate,
 			jetpackVersionSupportsLazyImages,
+			siteAcceleratorSupported,
+			photonModuleActive,
+			assetCdnModuleActive,
+			togglingSiteAccelerator,
 		} = this.props;
 		const isRequestingOrSaving = isRequestingSettings || isSavingSettings;
+		const cdnStatus = photonModuleActive || assetCdnModuleActive;
```

Comment:
This seems to be occurring at least two times. Perhaps it makes sense to include it in the props we return in `mapStateToProps`?

Reply:
Good call. I've done that.